packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.tsx

```diff
@@ -93,7 +103,7 @@ export class PaymentProvisioning {
     if (!this.authToken) console.error(missingAuthTokenMessage);
     if (!this.businessId) console.error(missingBusinessIdMessage);
 
-    this.refs = [this.coreInfoRef, this.legalAddressRef, this.additionalQuestionsRef, this.representativeRef, this.ownersRef];
+    this.refs = [this.coreInfoRef, this.legalAddressRef, this.additionalQuestionsRef, this.representativeRef, this.ownersRef, this.bankAccountRef];
     this.totalSteps = Object.keys(this.componentStepMapping).length - 1;
```

Comment:
If we use the getter as suggested above we could remove this line.

Reply:
Great catch - thanks man!